src/helper/functional.py

```diff
@@ -0,0 +1,8 @@
+class Functional:
+    @staticmethod
+    def flatten(xs):
+        return (xi for x in xs for xi in x)
+
+    @staticmethod
+    def unique(xs):
+        return list(set(xs))
```

Comment:
Is making a class necessary? Are there any reasons you chose to define as a class? From reading [this](https://stackoverflow.com/questions/19620498/how-to-create-a-utility-class-correctly) we prob can just put them as module-level functions.  (same goes to `paths_generator.py`) 

Remove the inheritance and just do sth like:
```
from ..helper.paths_generator import PathsGenerator as pg
from ..helper.functional import Functional as fct

fct.unique(fct.flatten(pg.generate_absolute_paths(blahblah)....
```

Wdyt??

Reply:
Agree. But the import would become more like `from ..helper import functional as fct`, since `Functional` would no longer be a class. Check the commited changes please to see if that's what you meant.

Reply:
Yea you right. LGTM.

Reply:
The change needs to be reflected in` tests/vision/test_detector.py` as well.

Reply:
Yea, regarding that; the tests concern solely the helper methods now.